PhysicsTools/PatAlgos/plugins/PATTriggerProducer.cc

```diff
@@ -157,7 +157,7 @@ PATTriggerProducer::PATTriggerProducer( const ParameterSet & iConfig ) :
 
   // HLT configuration parameters
   if ( iConfig.exists( "triggerResults" ) ) tagTriggerResults_ = iConfig.getParameter< InputTag >( "triggerResults" );
-  triggerResultsGetter_ = GetterOfProducts< TriggerResults >( InputTagMatch( InputTag( tagTriggerResults_.label(), tagTriggerResults_.instance() ) ), this);
+  triggerResultsGetter_ = GetterOfProducts< TriggerResults >( InputTagMatch( InputTag( tagTriggerResults_.label(), tagTriggerResults_.instance(), autoProcessName_ ? InputTag::kSkipCurrentProcess : nameProcess_  ) ), this);
```

Comment:
I do not think this works as intended. InputTagMatch does an exact match on the process name if one is given:
https://github.com/cms-sw/cmssw/blob/master/FWCore/Framework/interface/InputTagMatch.h#L38

I'm afraid no process name will ever match the string associated to `InputTag::kSkipCurrentProcess`.